src/reaper_osara.cpp

```diff
@@ -397,7 +409,12 @@ string formatTime(double time, TimeFormat timeFormat,
 			break;
 		}
 		case TF_FRAME: {
-			// Frames
+		case TF_ROUNDSEC: {
```

Comment:
I don't think this is meant to be immediately below case TF_FRAMES. That will break TF_FRAMES. :)